cfpb/cfgov-refresh
# Mega menu: Adjustments to featured links

Removes DS list classes, since more needed to be overridden than was being applied. It was only used for setting the left padding value to zero. 

## Removals

- Code comment to move menu-specific variable to DS, which doesn't make sense.

## Changes

- Removes DS list classes.
- Moves featured link icons into their associated links.

## Testing

1. Pull branch and build and visit homepage. 
2. Expand menu and visit featured links.
3. Resize page to see at responsive sizes.
4. compare to beta.consumerfinance.gov

## Screenshots

![hover](https://user-images.githubusercontent.com/704760/75196282-a9119980-5729-11ea-95f2-974eea8d1e49.gif)

## Todos

- External link utility needs adjusting to work again.
